src/adapters/postgres/cohort-adapter.ts

```diff
@@ -659,10 +644,17 @@ export class PostgresCohortService {
             HttpStatus.BAD_REQUEST
           );
         }
+        const order = {};
+        if (sort && sort.length === 2) {
```

Comment:
You should not check length like this.
        if (sort && sort.length) {
           order[sort[0]] = ['ASC', 'DESC''].inlcudes(sort[1].toUpperCase()) ? sort[1].toUpperCase() : 'ASC'
        } else {
        add by default ASC sorting on cohort names
        }

Reply:
@souravbhowmik1999